diesel/src/expression/expression_methods/global_expression_methods.rs

```diff
@@ -295,11 +295,10 @@ pub trait ExpressionMethods: Expression + Sized {
     ///     use self::posts::dsl::{posts, author_name};
     ///     let connection = establish_connection();
     ///
-    ///     let data = users.inner_join(posts)
+    ///     let data: QueryResult<Vec<String>> = users.inner_join(posts)
     ///         .filter(name.nullable().eq(author_name))
```

Comment:
Things that are likely to change for the same reason can be grouped (`.unwrap().collect()` for example)